.github/workflows/template_release_version.yml

```diff
@@ -0,0 +1,46 @@
+name: Release Version Detector
+
+on:
+  workflow_call:
+    outputs:
+      new_version:
+        value: ${{ jobs.new_version.outputs.new_version }}
+      new_tag:
+        value: ${{ jobs.new_version.outputs.new_tag }}
+
+jobs:
+  new_version:
+
+    name: Get new release version
+    runs-on: ubuntu-22.04
+
+    outputs:
+      new_version: ${{ steps.set_version.outputs.next_version }}
+      new_tag: ${{ steps.set_version.outputs.next_tag }}
+
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3.6.0
+        with:
+          fetch-depth: 0
+          fetch-tags: true
+      - name: Detect new release version
+        id: set_version
+        run: |
+          # fetch last tag from git
+          OLD_VERSION=`git describe --tags $(git rev-list --tags --max-count=1)`
+          PARTS=(${OLD_VERSION//./ })
```

Comment:
Do we have to strip the leading `v` here?

Reply:
It's not necessary because we don't need the year as variable in the workflow. 🙌

Reply:
Ah. 💡  Missed that one. Thank you! 🙏